test/check-kubernetes

```diff
@@ -186,19 +189,19 @@ class TestKubernetes(KubernetesCase):
         b.click(".details-listing tbody[data-id='nodes/127.0.0.1'] th")
         b.wait_present(".details-listing tbody[data-id='nodes/127.0.0.1'] tr.listing-panel")
```

Comment:
do we need a distinction here between present and visible? below, you changed the code to check for `is_visible`